modules/gsql-shell/pages/web.adoc

```diff
@@ -0,0 +1,57 @@
+= GSQL Shell (Web)
+:description: Introduction of the web version of GSQL shell.
+:experimental:
+
+TigerGraph provides the GSQL shell through the web as part of xref:master@home:ROOT:index.adoc#_tigergraph_suite[TigerGraph Suite].
+
+The web version of GSQL provides many user-friendly features such as command history, script uploading and downloading, keyboard shortcuts, autocomplete, syntax highlighting, and many more that make the GSQL shell easier to use for developers.
+
+== Supported browsers
+TigerGraph has verified support for the GSQL web shell on the following browsers:
```

Comment:
In other places where we verify browser support, we add the version. If that's not possible, can we say "up to date as of September 2022"?